system-int-test/test/cmr/system_int_test/search/humanizer/humanizer_test.clj

```diff
@@ -128,3 +134,68 @@
                                :body humanizers}]
 
       (is (= expected-humanizers (hu/get-humanizers))))))
+
+(deftest humanizer-report-test
+  (testing "Humanizer report saved successfully"
+    (let [humanizers (hu/make-humanizers)
+          ;; Ingest humanizers
+          _ (hu/update-humanizers transmit-config/mock-echo-system-token humanizers)
+          returned-humanizers (:body (hu/get-humanizers))
+          ;; sanity check
+          _ (is (= humanizers returned-humanizers))
```

Comment:
@mschmele Not sure what you mean? The `(is ...)` here does mean that when the tests are run, this will be treated as a test assertion ... and if it fails, the test will fail.